backend/src/tasks/hibp.ts

```diff
@@ -1,151 +0,0 @@
-import { Domain, Service, Vulnerability, connectToDatabase } from '../models';
-import { CommandOptions } from './ecs-client';
-import { plainToClass } from 'class-transformer';
-import saveVulnerabilitiesToDb from './helpers/saveVulnerabilitiesToDb';
-import axios from 'axios';
-
-/**
- * The hibp scan looks up emails from a particular .gov domain
- * that have shown up in breaches using the Have I
- * Been Pwned Enterprise API.
- * Be aware this scan can only query breaches from .gov domains
- */
-
-interface breachResults {
-  Name: string;
-  Title: string;
-  Domain: string;
-  BreachDate: string;
-  AddedDate: string;
-  ModifiedDate: string;
-  PwnCount: number;
-  Description: string;
-  LogoPath: string;
-  DataClasses: string[];
-  IsVerified: boolean;
-  IsFabricated: boolean;
-  IsSensitive: boolean;
-  IsRetired: boolean;
-  IsSpamList: boolean;
-  passwordIncluded: boolean;
-}
-
-async function getIps(organizationId?: String): Promise<Domain[]> {
-  await connectToDatabase();
-
-  let domains = Domain.createQueryBuilder('domain')
-    .leftJoinAndSelect('domain.organization', 'organization')
-    .andWhere('ip IS NOT NULL')
-    .andWhere('domain.name LIKE :gov', { gov: '%.gov' })
-    .andWhere('domain.ipOnly=:bool', { bool: false });
-
-  if (organizationId) {
-    domains = domains.andWhere('domain.organization=:org', {
-      org: organizationId
-    });
-  }
-
-  return domains.getMany();
-}
-
-async function lookupEmails(
-  breachesDict: { [key: string]: breachResults },
-  domain: Domain
-) {
-  try {
-    const { data } = await axios.get(
-      'https://haveibeenpwned.com/api/v2/enterprisesubscriber/domainsearch/' +
-        domain.name,
-      {
-        headers: {
-          Authorization: 'Bearer ' + process.env.HIBP_API_KEY!
```

Comment:
We should remove all secrets related to this (you can do a search in the codebase for HIBP_API_KEY) and remove things in terraform etc. Same thing with lookingglass.

Reply:
Done